columnflow/production/cms/btag.py

```diff
@@ -135,6 +140,35 @@ def add_weight(syst_name, syst_direction, column_name):
     return events
 
 
+def get_b_score_column(btag_config: tuple[str, list, str]) -> str:
+    """
+    Helper function to resolve the btag score column from the btag configuration.
+    """
```

Comment:
This docstring is not super helpful tbh. It would be better to also define the parameters and what is returned, e.g. 

```suggestion
    """
    Helper function to resolve the btag score column from the btag configuration.
    
    :param btag_config: Entry in auxiliary `config_inst.x.btag_sf`, see example :py:meth:`~columflow.production.cms.btag.btag_weights`. If tuple has less than 3 entries, the column name is derived from the name of the correction set.
    :returns: Name of column that is required for the calculation of this set of corrections.
    """
```